varats/varats/tools/tool_util.py

```diff
@@ -64,3 +66,22 @@ def get_research_tool(
 def get_supported_research_tool_names() -> tp.List[str]:
     """Returns a list of all supported research tools."""
     return ["phasar", "vara"]
+
+
+def configuration_lookup_error_handler(func: tp.Any) -> tp.Any:
+    """Wrapper for drivers to catch internal Exceptions and provide a helpful
+    message to the user."""
+
+    @wraps(func)
+    def wrapper_configuration_lookup_error_handler(
+        *args: tp.Any, **kwargs: tp.Any
+    ) -> tp.Callable[..., tp.Any]:
```

Comment:
My mistake. Either, we allow functions with return types to be wrapped, then we need to add a `return` in line 80, or we do not allow returns, then we need to do this change:
```suggestion
    ) -> tp.Callable[..., None]:
```